.npmignore

```diff
@@ -7,7 +7,6 @@ tests
 errorScreenshots
```

Comment:
Should we remove these?

https://github.com/cerner/terra-toolkit/blob/toolkit-v5-remove-nightwatch/tests/fixtures/nightwatch.html

https://github.com/cerner/terra-toolkit/blob/toolkit-v5-remove-nightwatch/tests/test.config.js#L10